### PR TITLE
Update Design & Add CFP Banner

### PR DESCRIPTION
**Pull request by matopher**
This updates the overall page design and adds a banner to promote the call for proposals.

### CODE REVIEW

File: site/includes/layouts/default.njk [hunk 1]
```diff
@@ -16,6 +16,8 @@
 
     {# favicon #}
     <link rel="icon" type="image/png" href="{{ site.images.favicon }}">
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@tailwindcss/ui@latest/dist/tailwind-ui.min.css">
```
Comment (matopher) [inline site/includes/layouts/default.njk]: I couldn't get the Tailwind UI plugin to play nice with our TailwindCSS setup. But once that's figured out, we can remove this.

**Review by maxxcrawford — CHANGES_REQUESTED** [review]
Logo tweaks. Looks great! 

Should we worry about the SASS files using the legacy `@tailwind` integration?

**Comment by matopher** [comment]
> Should we worry about the SASS files using the legacy `@tailwind` integration?

@maxxcrawford Good catch. Since we're only using TailwindCSS for styling, I updated the SASS directives and rebuilt our Tailwind config. Those changes seem to have done the trick!

I _think_ we should be good to go now.

**Review by kendallwerts — APPROVED** [review]
Looks great!

**Pull request merged**